guide/daily-spending-wallet/contacts.md

```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
 
 ### Importing an address
 
-In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the address and offers the user appropriate options. Here, the user adds the address as a new contact.
+This scenario can be initiated by copying a Lightning address to the clipboard, scanning a QR code, or tapping a payment link (see [payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }})). An address is passed into the application and, where it's recognized and appropriate options are shown to the user. In the example below, the user adds the address as a new contact.
 
 {% include image-gallery.html pages = page.imagesImportAddress %}
 
-This flow can also be initiated by scanning a QR code or tapping a payment link. For more, see [Sending bitcoin &middot; Payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }}).
-
 ### Importing an invoice
```

Comment:
```suggestion
### Importing a payment request
```

Payment request seems more fitting and aligns more with the requesting page. You also go back and forth between referring to it as an invoice and payment request in this section - bit confusing